tests/cli/workflow/mip/test_cli_mip_dna_run.py

```diff
@@ -5,26 +5,27 @@
 
 from cg.cli.workflow.mip_dna.base import run
 
-CASE_ID = "yellowhog"
-CONDA_ENV = "S_mip_rd-dna"
-EMAIL = "james.holden@scilifelab.se"
 
-
-def test_cg_dry_run(cli_runner, mip_context, caplog):
+def test_cg_dry_run(cli_runner, mip_context, caplog, case_id, email_adress):
     """Test print the MIP run to console"""
-    pedigree_path = mip_context.get("dna_api").get_pedigree_config_path(case_id="yellowhog")
+
+    caplog.set_level(logging.INFO)
+
+    pedigree_path = mip_context.get("dna_api").get_pedigree_config_path(case_id=case_id)
```

Comment:
Could we try to use Pthliv over os wherever possible? It is not your code originally, but we could replace os.link with Path.link_to()

Reply:
hmm, it seems like Path.link_to() requires python 3.8 and in prod we have 3.7.9 (according to the action test set-up). So it fails

Reply:
Should I revert to os.link again?

Reply:
Yes I have ended up in the same situation. We have to use `os.link` until we can update the CG env on both hasta and clinical (VMs) to python 3.8

Reply:
Alright! I have reverted to os.link for now